colcon_core/task/python/build.py

```diff
@@ -125,10 +126,9 @@ def __init__(self):  # noqa: D107
                 '--uninstall', '--editable',
                 '--build-directory', os.path.join(args.build_base, 'build')
             ]
-            rc = await check_call(
+            completed = await check_call(
                 self.context, cmd, cwd=args.build_base, env=env)
-            if rc:
-                return rc
+            return completed
```

Comment:
In this last case the variable `completed` isn't necessary anymore. Instead it can directly `return await check_call(...)`.